ipaserver/rpcserver.py

```diff
@@ -715,8 +715,7 @@ def listMethods(self, *params):
         """list methods for XML-RPC introspection"""
         if params:
             raise errors.ZeroArgumentError(name='system.listMethods')
-        return (tuple(unicode(cmd.name) for cmd in self.Command()
-                      if cmd is self.Command[cmd.name]) +
+        return (tuple(unicode(cmd.name) for cmd in self.Command) +
```

Comment:
This change and the ticket does not seem to go well together. What is the real issue reported by pylint here? I do not see it in the ticket referenced.

Reply:
I posted the list of errors pylint throws on Fedora 26. This particular patch is about the following:
`ipaserver/rpcserver.py:718: [E1102(not-callable), xmlserver.listMethods] self.Command is not callable)`

Reply:
Thanks. I think pylint is wrong here but I need to investigate more.

Reply:
So, this use of self.Command() comes from 79d1f5833547044a7cb2700454cacb2a0976dd5f which changed it all over the place. I think Pylint is definitely wrong here but we probably better to change `self.Command` to `self.api.Command` here to adhere logic of `plugable.Plugin` class which says `Command`, `env`, and `Backend` properties should be only used in a backward compatible code. Pylint gets it from a `@property` that self.Command returns something but it is unable to get inferred the type of the result (self.__api.Command) because we would only get that name space initialized runtime and thus gets it wrong.

Reply:
Also, client-side these are different objects:
```
>>> [i for i in api.Command][0]
<ipaclient.remote_plugins.schema._SchemaCommandPlugin object at 0x7f1e959d0a90>
>>> [i for i in api.Command()][0]
ipaclient.remote_plugins.schema.aci_add()
>>> 
```